### PR TITLE
Bug 1168328 - [Stingray][EPG] Add timeline progress bar

### PR DESCRIPTION
**Pull request by sean2449**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=55bb1de1504b531c4c686cd4153cbdc70fc7c4b6

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4e8bda0544580aebc94135905c75a872d5bde4ca

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f5082170bf947330c85dfd3c910eb7ca823aaecb

File: tv_apps/tv-epg/js/epg.js [hunk 1]
```diff
@@ -49,6 +46,10 @@
       this.visibleTimeOffset - this.visibleTimeSize,
       this.timelineUnit
     );
+
+    this.clock = new Clock();
```
Comment (huchengtw-moz) [inline tv_apps/tv-epg/js/epg.js]: What's the difference that you move clock code here? It looks that we don't need to do it.

Comment (sean2449) [inline tv_apps/tv-epg/js/epg.js]: The reason is to call clock.start after epgController is initiated.

**Pull request merged**